conf/flash_modes.xml

```diff
@@ -47,6 +47,7 @@
       <board name="li[s]?a_mx_.*"/>
       <board name="px4fmu_.*"/>
       <board name="elle*"/>
+      <board name="lia_.*"/>
```

Comment:
it should cover `lia_m_*` and `lia_s_*` boards... which one does it not cover?
But maybe it is easier adding them separately anyway...